pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -248,8 +248,8 @@ buildStdenv.mkDerivation ({
       $(< ${buildStdenv.cc}/nix-support/libc-cflags) \
       $(< ${buildStdenv.cc}/nix-support/cc-cflags) \
       $(< ${buildStdenv.cc}/nix-support/libcxx-cxxflags) \
-      ${lib.optionalString buildStdenv.cc.isClang "-idirafter ${buildStdenv.cc.cc}/lib/clang/${lib.getVersion buildStdenv.cc.cc}/include"} \
-      ${lib.optionalString buildStdenv.cc.isGNU "-isystem ${buildStdenv.cc.cc}/include/c++/${lib.getVersion buildStdenv.cc.cc} -isystem ${buildStdenv.cc.cc}/include/c++/${lib.getVersion buildStdenv.cc.cc}/${buildStdenv.hostPlatform.config}"} \
+      ${lib.optionalString buildStdenv.cc.isClang "-idirafter ${buildStdenv.cc.cc.lib}/lib/clang/${lib.getVersion buildStdenv.cc.cc}/include"} \
```

Comment:
since `cc` is defined as `libllvm.out`, the output is "predetermined" and stuff like `lib.getLib` won't work. it's a bummer but the only solution I can think of is to do "out" and "bin" instead of "out" and "lib" (i.e change the default output).

Reply:
That makes sense. I wonder if this behavior of `lib.get*` is actually necessary, but it is probably not a thing we should include here if it is at all possible since it would need additional testing.